Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -2674,6 +2671,7 @@ namespace AssetProcessor
                 AZ_TracePrintf(ConsoleChannel, "Builder optimization: %i / %i files required full analysis, %i sources found but not processed by anyone\n", m_numSourcesNeedingFullAnalysis, m_numTotalSourcesFound, m_numSourcesNotHandledByAnyBuilder);
             }
 
+            m_pathDependencyManager->ProcessQueuedDependencyResolves();
```

Comment:
So this file has a lot of whitespace changes, so I marking the actual changes.

This is the first out of two.
So the PathDependencyManager adds a call to process the dependency queue at this point